assets/main.scss

```diff
@@ -5,6 +5,16 @@
 
 @import "minima";
 
+@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro&display=swap');
```

Comment:
@huntertur Can we please start to look into moving some of these styles out of main.scss and into the _sass directory. Like I think the body style you added should be to a _base.scss and the header styles should be moved to a _header.scss. Then those files should be imported from this file. I think it would be good to start doing that sooner rather than later because otherwise this file will get cluttered with a bunch of random styles and it will make it consistent with the SSE webmaster.

Let me know if you need help with that.

Reply:
Hey Hunter, where's progress on this? Need any help?

Reply:
Was super busy the last week. Sorry ^^